fbgemm_gpu/docs/source/index.rst

```diff
@@ -0,0 +1,19 @@
+.. FBGEMM documentation master file, copied from fbgemm/docs
+   on Wed Jun 8 17:19:01 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to FBGEMM's documentation!
+=======================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+:ref:`genindex`
+
+Docs
+====
+
+.. doxygennamespace:: fbgemm
```

Comment:
```suggestion
.. toctree::
   :maxdepth: 2
   :caption: Contents

   fbgemm_gpu.fbgemm_gpu.rst

Reply:
Also add a file named fbgemm_gpu.fbgemm_gpu.rst and add the following to that file:

```
fbgemm_gpu.fbgemm_gpu
=====================

.. automodule:: fbgemm_gpu.fbgemm_gpu

fbgemm_gpu.fbgemm_gpu.batched_unary_embeddings_ops
--------------------------------------------------

.. automodule:: fbgemm_gpu.fbgemm_gpu.batched_unary_embeddings_ops
   :members:
   :show-inheritance:
```